app/jobs/inventory_host_updated_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Updates the host according to the inventory definition
+class InventoryHostUpdatedJob
+  include Sidekiq::Worker
+  include RecordNotFound
+
+  def perform(message)
+    return wrong_format_warning(message) unless valid_message_format(message)
+
+    rescue_not_found do
+      Host.find_by!(
+        id: message['host']['id']
+      ).update(name: message['host']['display_name'])
```

Comment:
Do you want this to silently fail?

```suggestion
      ).update!(name: message['host']['display_name'])
```